onnxruntime/python/tools/transformers/fusion_rotary_attention.py

```diff
@@ -363,6 +373,48 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             ["Transpose", "Reshape", "MatMul"],
             [1, 0, 0],
         )
+        _,v_nodes_4,_ = self.model.match_parent_paths_all(
+            matmul_qkv,
+            [
+                (
+                    ["Reshape", "Expand", "Unsqueeze", "Concat", "Transpose", "Reshape", "MatMul"],
```

Comment:
Why the exported model has so many patterns for rotary?

Reply:
It is intended to match a sub-graph for this repeat_kv pattern. The sub-graph looks like below:
![repeak_kv](https://github.com/microsoft/onnxruntime/assets/123416088/7c42a39b-b227-4452-a0b9-02ff5f5ae25a)